core/embed/rust/src/ui/model_mercury/flow/confirm_reset_create.rs

```diff
@@ -61,7 +61,7 @@ impl FlowState for ConfirmResetCreate {
 
 use crate::{
     micropython::{map::Map, obj::Obj, util},
-    ui::layout::obj::LayoutObj,
+    ui::{layout::obj::LayoutObj, model_mercury::component::HoldToConfirm},
```

Comment:
Can we put the `HoldToConfirm` to the `super::super` part

Reply:
reorganized imports https://github.com/trezor/trezor-firmware/pull/3776/commits/834ede0d3f37cd4d9d10e1d5a89626cef78385b6